core/src/main/scala/org/apache/pekko/persistence/jdbc/query/javadsl/JdbcReadJournal.scala

```diff
@@ -32,7 +38,8 @@ class JdbcReadJournal(journal: ScalaJdbcReadJournal)
     with CurrentEventsByPersistenceIdQuery
     with EventsByPersistenceIdQuery
     with CurrentEventsByTagQuery
-    with EventsByTagQuery {
+    with EventsByTagQuery
+    with CurrentLastKnownSequenceNumberByPersistenceIdQuery {
```

Comment:
Do we really need the CurrentLastKnownSequenceNumberByPersistenceIdQuery trait? Can't we just add the function without having a marker trait.
We can add the trait to the pekko-persistence jar at some stage and then do another PR to uptake the trait here.

Reply:
My aim would be merge this without the marker trait meaning that we could release a version pekko-persistence-jdbc 1.1.x that supports this but without needing a new Pekko core release. pekko-persistence 1.2.0-M2 and above could have the marker trait and a pekko-persistence-jdbc 1.2.0(-M1) release could rely on  pekko-persistence 1.2.0-M2 and uptake the marker trait.

I haven't studied pekko-persistence-r2dbc but I presume that it too could implement this support and in a similar timeline.

Reply:
Sure, I will drop the marker trait today or tomorrow.

Reply:
Removed marker trait with <https://github.com/apache/pekko-persistence-jdbc/pull/267/commits/ce15d5ea2627de79e32c025b1caa971d5d761320>.

Reply:
@ptrdom it looks like pekko-persistence-r2dbc can have a similar PR added at some stage? Can you confirm this? I can look into adding the marker trait to the pekko core but it could be a while before it makes it into a full release meaning that we will have to wait to add the uptake of the marker trait.

Reply:
Yes, I think it should be no problem to add this functionality to pekko-persistence-r2dbc.